deploy/healthCheck.sh

```
@@ -0,0 +1,29 @@
+  #!/bin/bash
```

Shellcheck (0.10.0)

[error] Remove leading spaces before the shebang.

(SC1114)